### PR TITLE
AAP-16453 - Minor post-docathon fixes

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR addresses minor issues found checking the post-docathon output in Pantheon for the following:

Getting started with automation hub
Managing content in automation hub
Ansible Automation Platform installation guide

### CODE REVIEW

File: downstream/assemblies/dev-guide/assembly-tools-components.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ include::core/con-about-ansible-cli.adoc[leveloffset=+1]
 == Additional resources
 
 * For more information on how to use Ansible as a command line tool, refer to link:https://docs.ansible.com/ansible/latest/command_guide/command_line_tools.html[Working with command line tools] in the Ansible _User Guide_.
-* To upload content to {HubName}, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/getting_started_with_automation_hub/index#uploading_content_to_red_hat_automation_hub[Uploading content to automation hub] in the {PlatformNameShort} product documentation.
+* To upload content to {HubName}, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/getting_started_with_automation_hub/uploading_content_hub[Uploading content to automation hub] in the {PlatformNameShort} product documentation.
```
Comment (hherbly) [inline downstream/assemblies/dev-guide/assembly-tools-components.adoc:32]: I tried opening this updated link (with 2.4 where PlatformVers is) but it still seems to be broken.

Comment (dcdacosta) [reply downstream/assemblies/dev-guide/assembly-tools-components.adoc]: Link needed to be uploading-content-hub with dashes and not underscores. This is fixed now.

File: downstream/modules/platform/ref-hub-configs.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ See the following resources to explore additional {HubName} configurations.
...
-|link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_content_in_automation_hub/index#assembly-managing-private-collections[Publishing proprietary content collections in {HubName}]|Publish internally developed collections on your {HubName}
+|link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/getting_started_with_automation_hub/assembly-user-access[Managing user access in private {HubName}]|Configure user access for {HubName}
+|link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_content_in_automation_hub/managing-cert-valid-content[Managing Red Hat Certified, validated, and Ansible Galaxy content in {HubName}]|Add content to your {HubName}
+|link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_content_in_automation_hub/assembly-managing-private-collections[Publishing proprietary content collections in {HubName}]|Publish internally developed collections on your {HubName}
```
Comment (hherbly) [inline downstream/modules/platform/ref-hub-configs.adoc:13]: The last listed link here also seems to be broken.

**Review by hherbly — APPROVED** [review]
LGTM!

**Pull request merged**